proposals/lip-Introduce_cross-chain_update_mechanism.md

```diff
@@ -0,0 +1,620 @@
+   The `isLive` function is specified in [LIP "Introduce Interoperability module"][base-interoperability-LIP], and `timestamp` is the timestamp of the block including the CCU.
+
+
+##### Liveness Requirement for the First CCU
+
+If `sendingChain.status == CHAIN_REGISTERED`, the proposed CCU must contain a non-empty `certificate` which must follow the schema defined in  [LIP "Introduce a certificate generation mechanism"][certificate-generation-LIP]. 
+In the following, let `certificate` be the deserialized certificate.
+
+Furthermore, the certificate is only valid if it allows the sidechain account to remain live for a reasonable amount of time.
+This is done by checking that
+```java
+timestamp - CCU.params.certificate.timestamp < 15*24*3600
+```
+where `timestamp` is the timestamp of the block including the CCU.
+
+
+#### Certificate and Validators Update Validity
+
+If `params` contains a non-empty `certificate`, it is valid if:
+
+*  `params.certificate` follows the schema defined in [LIP "Introduce a certificate generation mechanism"][certificate-generation-LIP]. 
+   Again, we write `certificate` for the deserialized certificate.
+*  `certificate.height` is greater than `sendingAccount.lastCertifiedHeight`.
+*  `certificate.timestamp < timestamp`, where timestamp is the timestamp of the block including the CCU.
+*  `certificate.signature` is a valid aggregate signature for the `sendingAccount.validators` property of the cross-chain account. This is done by verifying that the function below defined in the [LIP 0038][BLS-LIP-verifyWeightedAggSig] returns `VALID`. 	
+	```java
+	verifyWeightedAggSig( 
+	    keysList          = sendingAccount.validators.keys,  
+	    aggregationBits.  = certificate.aggregationBits,  
+	    signature         = certificate.signature,  
+	    tag               = MESSAGE_TAG_CERTIFICATE, 
+	    networkIdentifier = sendingAccount.networkID,  
+	    weights           = sendingAccount.validators.weights,  
+	    threshold         = sendingAccount.validators.threshold, 
+	    message           = params.certificate 
+	)
+	```
+
+If `params` contains a non-empty `validatorsUpdate` property (with deserialized value not equal to `{keysUpdate:[], weightsUpdate:[], newCertificateThreshold: 0}`), 
+it is valid if:
+
+*   `params` contains a non-empty `certificate`.
+*   `validatorsUpdate` has the correct format:
+    *   for all `activeValidator` in `validatorsUpdate.activeValidators` we have that `activeValidator.blsKey` is a BLS public keys, hence are 48 bytes long.
+    *   every `activeValidator` in `validatorsUpdate.activeValidators` has a different `activeValidator.blsKey`.
+*   `certificate.validatorsHash` is obtained as the SHA-256 digest of the updated `sendingAccount.validators`, see ["Update Validators" section](#update-validators) below, serialized according to `chainAccountSchema.validators` defined in the [LIP "Introduce Interoperability module"][base-interoperability-LIP] (and copied below).
```

Comment:
I think the idea could be to instead call the API from the BFT module (just a reminder, this is still not decided)

Reply:
done. Will update if name changes.